cmd/server-mux_test.go

```diff
@@ -109,20 +109,18 @@ func dial(addr string) error {
 
 // Tests initializing listeners.
 func TestInitListeners(t *testing.T) {
-	portTest1 := getFreePort()
-	portTest2 := getFreePort()
```

Comment:
GetFreePort uses port '0' to get the free port. GetFreePort should work appropriately in the first place did you investigate why it doesn't work ?